server/service/src/sync/translations/store.rs

```diff
@@ -38,11 +38,10 @@ impl SyncTranslation for StoreTranslation {
         // HIS -> Hospital Information System (no name_id)
         // SM -> Supervisor Store
         // DRG -> Drug Registration (name_id exists but no name with that id)
-        match &data.code[..] {
-            "HIS" => return Ok(None),
-            "DRG" => return Ok(None),
-            "SM" => return Ok(None),
-            _ => {}
+        // TODO: Ideally we want another state, `Ignored`
+        // (i.e. return type) Translation Not Matches, Translation Ignored (with message ?) and Translated records
+        if let "HIS" | "DRG" | "SM" = &data.code[..] {
```

Comment:
oh new Rust syntax or was that always in?

Reply:
not sure when it was added, i had a look at [this](https://doc.rust-lang.org/book/ch18-03-pattern-syntax.html) awhile ago and remembered the `|` when touching this file, sorry I only changed it since I was returning empty array and it was easier to combine the match statements, but reverted the change, and kept the syntax and added comment